source/dhtslib/util.d

```diff
@@ -0,0 +1,51 @@
+module dhtslib.util;
+
+import std.string : toStringz;
+import std.typecons : tuple;
+
+import dhtslib.coordinates;
+import htslib.sam : bam_name2id;
+import htslib.vcf : bcf_hdr_id2name;
+import htslib.hts : hts_parse_reg64;
+
+/// Returns tuple of String and ZBHO coordinates
+/// representing input string. Supports htslib coordinate strings.
+/// i.e chr1:1-10
+auto getIntervalFromString(string region){
+    ZBHO coords;

```

Comment:
Coordinates written in this form are typically 1-based AFAIK

Reply:
`hts_parse_reg64` accepts strings that represent 1-based intervals, however this is returned as ZBHO.